src/Take.Blip.Client/Extensions/Contacts/IContactExtension.cs

```diff
@@ -76,8 +76,14 @@ public static class ContactExtensionExtensions
                 
                 contact = new Contact
                 {
-                    Identity = identity, Name = account.FullName
+                    Identity = identity,
+                    Name = account.FullName
                 };
+
+                if (account.Source != String.Empty)
```

Comment:
Use the string primitive type instead of the System.String struct.
```suggestion
                if (account.Source != string.Empty)
```